### PR TITLE
Misc fallback program fixes from Endless OS

### PR DESCRIPTION
**Pull request by jprvita**
These are a few misc fixes and improvements to the fallback program that we have been carrying downstream at Endless OS.

These changes we already reviewed twice via the community review process on https://github.com/rhboot/shim-review/issues/61 and https://github.com/rhboot/shim-review/issues/105.

### CODE REVIEW

File: fallback.c [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 #include <efi.h>
 #include <efilib.h>
+#include <CrtLibSupport.h>
```
Comment (vathpela) [inline fallback.c:9]: This looks incorrect, and it is breaking the CI build - but just removing it works fine.

**Review by vathpela — CHANGES_REQUESTED** [review]
Aside from the one minor issue, it looks good to me.

**Comment by ManMower** [comment]
FWIW, this series LGTM

File: fallback.c (hunk 1 above)
Comment (jprvita) [reply fallback.c]: Looks like we need `OpenSslSupport.h` there, for `isxdigit`.

**Comment by vathpela** [comment]
I had meant that I would pull it after the other changeset, which also obviated the need to add the second include, but oh well ;)

Anyway, pushed as 9bfdf2dea93621c9231793e2ade6b56abf7f6b8d.

**Pull request closed without merging**